enzy_htp/structure/ligand.py

```diff
@@ -35,15 +35,38 @@ class Ligand(Residue):
 
     Attributes:
         net_charge : The net charge of the molecule as an int.
+        bonds: 
+        conformer_coords:
     """
 
     def __init__(self, residue_idx: int, residue_name: str, atoms: List[Atom], parent=None, **kwargs):
         """
         Constructor for Ligand. Identical to Residue() ctor but also takes net_charge value.
         """
+        self.bonds = kwargs.get('bonds', list()) 
         self.net_charge = kwargs.get("net_charge", None)
         Residue.__init__(self, residue_idx, residue_name, atoms, parent)
         self.rtype = chem.ResidueType.LIGAND
+        self.conformer_coords = list()
```

Comment:
In my original design conformations is a seperate class like StructureEnsemble for Structure(). Can we make a class called LigandEnsemble or LigandConformers and is returned by generate_conformers(). 

The reason I make this argument is I want to avoid implicit data which make it a bit hard for writing interfaces. (i.e.: "a function that requires LigandConformers" vs "a function that requires Ligand with an implicit attribute conformer_coords")